satpy/enhancements/__init__.py

```diff
@@ -106,6 +113,7 @@ def apply_enhancement(data, func, exclude=None, separate=False,
 def crefl_scaling(img, **kwargs):
     """Apply non-linear stretch used by CREFL-based RGBs."""
     LOG.debug("Applying the crefl_scaling")
+    warnings.warn("'crefl_scaling' is deprecated, use 'interp_scaling' instead.", DeprecationWarning)
 
     def func(band_data, index=None):
```

Comment:
could the rest of this function be replace with `interp_scaling`?

Reply:
Yes, of course. :man_facepalming: